src/librustc/session/config.rs

```diff
@@ -1591,6 +1591,17 @@ pub fn default_configuration(sess: &Session) -> ast::CrateConfig {
     if sess.opts.crate_types.contains(&CrateType::ProcMacro) {
         ret.insert((sym::proc_macro, None));
     }
+    match sess.opts.cg.tracer {
+        TracerMode::Software => {
+            ret.insert((Symbol::intern("tracermode"), Some(Symbol::intern("sw"))));
```

Comment:
Maybe prefix with `yk_`?

Reply:
I tried to make it similar to the command line arguments that invoke the tracer, e.g. `-C tracer=hw`. So if we were to add `yk_` we should add that to the compiler argument as well. Alternatively, maybe we should also rename `tracermode` to `tracer` so it's the same as the compiler flag?

Reply:
We'd probably also have to change `#[no_swt]`. Let's roll without the prefix for now, and if things get confusing at any point, we can add the prefix.

Thanks for the suggestion Bjorn!